lib/stac-auth-proxy/runtime/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG PYTHON_VERSION
+FROM --platform=linux/amd64 public.ecr.aws/lambda/python:${PYTHON_VERSION}
+
+WORKDIR /tmp
+RUN python -m pip install pip -U
+
+COPY stac-auth-proxy/runtime/requirements.txt requirements.txt
+RUN python -m pip install -r requirements.txt "mangum>=0.14,<0.15" -t /asset
```

Comment:
Feels a little strange to specify dependencies here _and_ in `requirements.txt`?